tests/integration/golang/database/mlflow.sql

```diff
@@ -0,0 +1,161 @@
+PRAGMA foreign_keys=OFF;
```

Comment:
there is no mention of the MLFlow version that this schema comes from. I'm sorry but this is a really poor implementation, and I would much rather have a proper test that can create databases from various versions of MLFlow (and cache them) - we could easily do this with e.g. dagger so that it's cached and parallelised

Reply:
would adding a version help this? I don't think the perfect should be the enemy of the good here, and the perfect will take some time to develop.

Reply:
adding a version would be the bare minimum, yes — and if you go for just 1 version on 1 backend, please at least use the oldest version of MLFlow we support so that the test is relevant

Reply:
now labeled and testing oldest supported and most recent MLflow schema

Reply:
Reviewed requested change, LGTM